upup/models/cloudup/resources/addons/networking.projectcalico.org/pre-k8s-1.6.yaml.template

```diff
@@ -70,6 +70,8 @@ spec:
       labels:
         k8s-app: calico-node
         role.kubernetes.io/networking: "1"
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
So before 1.6, daemonsets ignored tolerations.  So we don't need the toleration annotation.

It is harmless though...